app/models/events/new_discussion.rb

```diff
@@ -1,7 +1,8 @@
 class Events::NewDiscussion < Event
   def self.publish!(discussion)
     event = create!(kind: 'new_discussion',
-                    eventable: discussion)
+                    eventable: discussion,
+                    sequence_id: 0)
```

Comment:
Thanks for this.. However I'm still confused. My reasoning behind not giving new_discussion events a discussion_id was that they don't show up in the discussion thread.

IE: The rule has been that only event which are supposed to show up in the discussion thread are sequenced.

My question is: how is the new_discussion event related to the bug you're fixing?